packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -309,32 +311,111 @@ tbody.spectrum-Table-body {
 }
 
 .spectrum-Table-colResizeIndicator {
+  --spectrum-table-col-resize-indicator-background-color: var(--spectrum-global-color-blue-600);
   &.spectrum-Table-colResizeIndicator--resizing {
     background-color: var(--spectrum-table-col-resize-indicator-background-color);
   }
 }
 .spectrum-Table-colResizeNubbin {
+  --spectrum-table-col-resize-nubbin-background-color: var(--spectrum-global-color-blue-600);
+  --spectrum-table-col-resize-nubbin-icon-color: var(--spectrum-global-color-static-white);
   svg {
-    > g {
+    g[fill][stroke] {
       fill: var(--spectrum-table-col-resize-nubbin-background-color);
       stroke: var(--spectrum-table-col-resize-nubbin-background-color);
     }
-    > path {
+    path[fill][stroke] {
       fill: var(--spectrum-table-col-resize-nubbin-icon-color);
       stroke: var(--spectrum-table-col-resize-nubbin-icon-color);
     }
   }
 }
 .spectrum-Table-bodyResizeIndicator {
+  --spectrum-table-body-resize-indicator-background-color: var(--spectrum-global-color-blue-600);
   background-color: var(--spectrum-table-body-resize-indicator-background-color);
 }
 
 @media (forced-colors: active) {
-  .spectrum-Table {
+  .spectrum-Table.spectrum-Table {
     --spectrum-table-col-resize-indicator-background-color: Highlight;
     --spectrum-table-col-resize-nubbin-background-color: Highlight;
     --spectrum-table-col-resize-nubbin-icon-color: HighlightText;
     --spectrum-table-body-resize-indicator-background-color: Highlight;
     --spectrum-table-divider-border-color: CanvasText;
+    --spectrum-table-cell-border-color-key-focus: Highlight;
+    --spectrum-table-background-color: Canvas;
+    --spectrum-table-row-background-color-hover: Canvas;
+    --spectrum-table-row-background-color-down: Canvas;
+    --spectrum-selectlist-option-focus-indicator-color: Highlight;
+    --spectrum-table-cell-border-color: CanvasText;
+    --spectrum-table-row-border-color-selected: Highlight;
+    --spectrum-alias-border-color-focus: Highlight;
+
+    .spectrum-Table-colResizeIndicator {
+      --spectrum-table-col-resize-indicator-background-color: Highlight;
+    }
+
+    .spectrum-Table-colResizeNubbin {
+      --spectrum-table-col-resize-nubbin-background-color: Highlight;
+      --spectrum-table-col-resize-nubbin-icon-color: HighlightText;
+    }
+
+    .spectrum-Table-bodyResizeIndicator {
+      --spectrum-table-body-resize-indicator-background-color: Highlight;
+    }
+  }
+
+  .spectrum-Table-headWrapper {
+    forced-color-adjust: none;
```

Comment:
With `forced-colors: active`, borders are rendered using the foreground color, `CanvasText`, provided but the operating system, even when they are `transparent`.  We use transparent borders on the `.spectrum-Table-headWrapper` above, probably to ensure that text within the column headers aligns with the text in the table body: https://github.com/adobe/react-spectrum/pull/3827/files#diff-c4ea0189e6859fb88727a36a4d2b2ad1ca2b989a76b2789447c7e7d1fd6affadR25-R28.
```css
.spectrum-Table-headWrapper {
  border-inline-start-color: transparent;
  border-inline-end-color: transparent;
}
```
So that `forced-color-adjust: none;` does not override `forced-colors: active` for the header row and cells contained within `.spectrum-Table-headWrapper`, I add `forced-color-adjust: none;` to the `.spectrum-Table-headCell`.